src/utils/composeStores.js

```diff
@@ -1,11 +1,18 @@
-import mapValues from '../utils/mapValues';
-import pick from '../utils/pick';
+import mapValues from './mapValues';
+import pick from './pick';
+
+// UNDEF = undefined;
+const UNDEF = void 0;
```

Comment:
I thought about that. I was trying to make the test as obvious as possible. I will change it to above